### PR TITLE
Transportaion cargo

### PR DESCRIPTION
**Pull request by shayIzhak2003**

### CODE REVIEW

File: c-sharp-apps-shay-shalom-izhak/transportaion-Cargo-App/RunTranportaionApp2.cs [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+            // Load the table onto the train
+            bool trainLoadResult4 = testTrain.Load(tableItem);
+            DisplayLoadResult(trainLoadResult4, "Train", "Table Item");
+
+            // Load the sofa onto the train
+            bool trainLoadResult5 = testTrain.Load(sofaItem);
+            DisplayLoadResult(trainLoadResult5, "Train", "Sofa Item");
+
+            // Travel to Port 2
+            testTrain.DistanceToNextPort = 1500;
```
Comment (yim222) [inline c-sharp-apps-shay-shalom-izhak/transportaion-Cargo-App/RunTranportaionApp2.cs:107]: Try to add here this code and see how you get error. 
I believe you can suggest some idea's from here regarding design patterns

` Console.WriteLine("lingar - " + testTrain.GetPricingList());`

File: c-sharp-apps-shay-shalom-izhak/transportaion-Cargo-App/Train.cs [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace c_sharp_apps_shay_shalom_izhak.transportaion_Cargo_App
+{
+    public class Train : CargoVehicle
+    {
+        private Cron cargoLimiter;
+
```
Comment (yim222) [inline c-sharp-apps-shay-shalom-izhak/transportaion-Cargo-App/Train.cs:12]: I am expecting to see here array or list not single Cron (like in the real world). 
At ship too - multiple Containers.

**Review by yim222 — COMMENTED** [review]
היי ראה הודעה בסלאק

**Comment by yim222** [comment]
Approved

**Comment by yim222** [comment]
Approved, Great Work

File: c-sharp-apps-shay-shalom-izhak/transportaion-Cargo-App/RunTranportaionApp2.cs (hunk 1 above)
Comment (shayIzhak2003) [reply c-sharp-apps-shay-shalom-izhak/transportaion-Cargo-App/RunTranportaionApp2.cs:107]: hi izhar , how are you ? look i have just checked that block of code and there wesent any errors the output was (Train successfully loaded Sofa Item.)

**Pull request merged**